dockerfiles/frameworks/src/Dockerfile

```diff
@@ -163,5 +163,26 @@ ADD flow/patch.patch ./
 RUN set -xe; \
     git apply *.patch
 
-
 CMD [ "./run.sh" ]
+
+FROM base AS phpredis
+ARG REDIS_VERSION_TAG=5.0.1
```

Comment:
We support `"predis/predis": "^1.1"`, can we test that version. In general an approach would be any new major we support?